src/plugins/timeframe-selector-plugin.ts

```diff
@@ -300,19 +300,16 @@ export default class TimeframeSelectorPlugin {
         this.dots = dots.sort((a, b) => {
             if (a.pos !== b.pos) {
                 return a.pos - b.pos;
-            } else {
```

Comment:
Sometimes I wrote unnecessarily complicated code.(

Reply:
me too, luckily linters are really good for such things